docs/accounts/remove.md

```diff
@@ -2,9 +2,17 @@
 
 To remove an email account from K-9 Mail do the following:
 
-1. Go to the [accounts screen](view.md)
-1. Long-press the name of the account you want to remove. A menu pops up.
-1. Select the option "Remove account"
-1. You're asked to confirm the removal of the account. Click "OK" to confirm.
+Open the settings screen:
 
-![Account options - delete](img/account_remove_account.png)
+![settings icon](../img/screenshots/settings_select.png)
+
+Select the menu icon in the top corner:
+
+![menu icon](../img/screenshots/account_menu_icon.png)
+
```

Comment:
This is missing instructions to select an account to go to the account settings screen.

Reply:
fixed